src/App.tsx

```diff
@@ -20,12 +23,25 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={filmName}
+                onChange={(event) => {
+                  setFilmName(event.target.value);
+                  setVisibleMovies(moviesFromServer.filter(movie => {
+                    return movie
+                      .title
+                      .toLowerCase()
+                      .includes(filmName.toLowerCase()) || movie
+                      .description
+                      .toLowerCase()
+                      .includes(filmName.toLowerCase());
+                  }));
+                }}
```

Comment:
move this to separate callback and split code into variables and functions to improve readability